libs/search/src/lib/catalog-title/catalog-title.component.html

```diff
@@ -0,0 +1,5 @@
+<h1 class="p-5 text-3xl font-bold"
```

Comment:
should be a dumb component in `lib-ui` lib, that takes `name `and `org` as inputs

Reply:
I understand why you have difficulties on finding component name - We are going to be short on naming with all these components ...

Reply:
Dumb one created.